src/main/webapp/app/exercises/modeling/manage/modeling-exercise-detail.component.ts

```diff
@@ -54,6 +57,12 @@ export class ModelingExerciseDetailComponent implements OnInit, OnDestroy {
             this.load(params['exerciseId']);
         });
         this.registerChangeInModelingExercises();
+        // Checks if the current environment includes "apollon" profile
+        this.profileService.getProfileInfo().subscribe((profileInfo) => {
+            if (profileInfo && profileInfo.activeProfiles.includes('apollon')) {
+                this.isApollonProfileActive = true;
+            }
```

Comment:
Would it make sense to simplify this to a one-liner like `this.isApollonProfileActive = !!(profileInfo && ...);`?
Or does it make this section less readable? What do you think?

Reply:
I think that it doesn't matter much